### PR TITLE
Toxic Exposure: Add basic toxicExposure node to pre-submission JSON

### PR DESCRIPTION
**Pull request by sethdarragile6**
This PR adds the top-level `toxicExposure` JSON node to the pre-submission JSON. At this stage, this is merely a pass-through. Later, it will be transformed by the Form526ToLighthouseTransformer into the Lighthouse request format

## Summary

- *This work is behind a feature toggle (flipper): YES (disability_526_toxic_exposure)*
- *Disability Benefits Experience - Team 1 (DBEX-TREX🦖)*

## Related issue(s)

- *Toxic Exposure - Data Transformation #2 - Save TE fields for future transformation#[79935](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/79935)*

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
     Pre-submission form json minus the new toxicExposure node would NOT be added to the submission form json
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
     With the disability_526_toxic_exposure flag ON, submit a 526ez with TE questions answered, confirm that the TE nodes make it to the saved submission record
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: spec/support/disability_compensation_form/submissions/with_everything.json [hunk 1]
```diff
@@ -191,7 +191,69 @@
...
+            "startDate": "1991-03-01",
+            "endDate": "1992-01-01"
+          },
+          "qatar": {
+            "startDate": "1991-02-12",
+            "endDate": "1991-06-01"
+          },
+          "kuwait": {
+            "startDate": "1991-03-15"
+          }
```
Comment (freeheeling) [inline spec/support/disability_compensation_form/submissions/with_everything.json:212]: Will not providing an `endDate` for Kuwait or Laos potentially be an issue in the future?

File: spec/support/disability_compensation_form/submissions/with_everything.json [hunk 2]
```diff
@@ -191,7 +191,69 @@
...
+            "startDate": "1991-03-01",
+            "endDate": "1992-01-01"
+          }
+        },
+        "specifyOtherExposures": {
+          "description": "Lead, burn pits",
+          "startDate": "1991-03-01",
+          "endDate": "1992-01-01"
+        },
+        "toxicExposureConditions": {
```
Comment (christinec-fftc) [inline spec/support/disability_compensation_form/submissions/with_everything.json:253]: this is getting renamed to just `conditions` in https://github.com/department-of-veterans-affairs/vets-website/pull/29344. Do we need to account for this now or update later?

File: spec/support/disability_compensation_form/submissions/with_everything.json (hunk 1 above)
Comment (christinec-fftc) [reply spec/support/disability_compensation_form/submissions/with_everything.json:212]: each startDate or endDate is optional for Veterans and nullable in LH afaik

Comment (sethdarragile6) [reply spec/support/disability_compensation_form/submissions/with_everything.json:212]: No, in the case of these date ranges, both the start and the end dates are optional

File: spec/support/disability_compensation_form/submissions/with_everything.json (hunk 2 above)
Comment (sethdarragile6) [reply spec/support/disability_compensation_form/submissions/with_everything.json:253]: that's a good call-out. probably would wind up changing it this sprint, but we might as well update it now

**Review by aurora-a-k-a-lightning — APPROVED** [review]

File: spec/support/disability_compensation_form/submissions/with_everything.json (hunk 1 above)
Comment (sethdarragile6) [reply spec/support/disability_compensation_form/submissions/with_everything.json:212]: uh, what Christine said 😂. I mustn't've hit refresh on the PR before commenting

**Pull request merged**